docker/server/docker-compose.yml.example

```diff
@@ -0,0 +1,15 @@
+version: '3.3'
+services:
+  kiwix-serve:
+      ports:
+          - 8080:80
+      image: kiwix/kiwix-serve
+      # uncomment next 5 lines to use it with local zim file in /tmp/zim
+      # volumes:
+      #   - /tmp/zim:/data
+      # command:
+      #   - wikipedia.zim
+      #   - wiktionary.zim
+      # uncomment next 2 lines to use it with remote zim file
+      # environment:
+      #   - 'DOWNLOAD=https://download.kiwix.org/zim/wikipedia_bm_all.zim'
```

Comment:
@rgaudin @Teraskull is it possible to pass multiple URL via DOWNLOAD variable? if not, how does one use multiple zim file while using remote download function?

Reply:
No, it is limited to one ZIM file.

Reply:
Have a look to the pretty simple `start.sh` file to understand how it works https://github.com/kiwix/kiwix-tools/blob/master/docker/server/start.sh. You can still specify whatever you want as argument (so a custom `library.xml` for example.